.github/workflows/self-release.yml

```
@@ -0,0 +1,37 @@
+
+permissions:
+  id-token: write
+  contents: write
+  pull-requests: write
+  packages: read
+
+jobs:
+  publish-release:
+    uses: ./.github/workflows/release.yml
+    with:
+      runner: ubuntu-24.04
```

actionlint (1.7.9)

input "runner" is not defined in "./.github/workflows/release.yml" reusable workflow. defined inputs are "filter_paths", "path_level", "runner_type", "semantic_version"

(workflow-call)